src/views/datasets/DatasetDetailView.vue

```diff
@@ -258,6 +269,20 @@ watch(
               <h2 class="fr-mb-1v subtitle subtitle--uppercase">
                 {{ typedResources.typeLabel }}
               </h2>
+              <DsfrSearchBar
+                button-text="Rechercher"
+                placeholder="Rechercher"
+                :large="false"
+                class="search-bar"
+                @search="doSearch"
+                @update:modelValue="updateQuery"
+                v-if="
+                  typedResources.typeLabel != 'Documentation' &&
+                  (typedResources.total > 6 ||
+                    (typedResources.total <= 6 && query != ''))
```

Comment:
```suggestion
                  (typedResources.total > 6 || query !== '')
```

Je suis fatigué mais ça me semble simplifiable comme ça ?

Reply:
Also le 6 en conf ce serait top 🙏

Reply:
Existe en conf => website.pagination_sizes.files_list

Reply:
le 6 est bien paramétrable ✅ 
Pour la simplification j'ai essayé ta méthode cela ne marche pas car ça fait disparaitre la barre de recherche si il y a moins de 6 ressources, même lorsque nous sommes en cours de filtrage.